web/src/components/DandiAppBar.vue

```diff
@@ -81,13 +126,25 @@
 </template>
 
 <script>
-import { mapActions, mapGetters } from 'vuex';
+import { stringify } from 'qs';
+import { mapActions, mapGetters, mapState } from 'vuex';
 import { Search as GirderSearch, Authentication as GirderAuth } from '@girder/components/src/components';
 
 export default {
+   data: () => ({
+    name: '',
+    description: '',
+    regdialog: false,
+  }),
   components: { GirderSearch, GirderAuth },
   computed: {
+    saveDisabled() {
+      return this.name === "" || this.description === "";
```

Comment:
More concise syntax
```suggestion
      return !(this.name && this.description);
```

Reply:
I personally prefer the `A + B` form over it's de Morgan's complement -- I think it reads better.

Reply:
So `!this.name || !this.description`?